README.md

```diff
@@ -1 +1,40 @@
 This is a demo app for [remoteStorage.js](http://remotestorage.io/integrate/).
+
+# Quick Start
+
+[1] Visit https://myfavoritedrinks.remotestorage.io/
+
+* it's this app hosted
+* the app needs to be hosted using a Web server--don't just run [index.html](index.html) from file system
+
+[2] Start a remotestorage server
+
+* there are [many options](https://wiki.remotestorage.io/Servers)
+
+> If using node/npm ecosystem:
+>
+> 1. clone https://github.com/remotestorage/armadietto
+> 1. cd $armadietto
+> 1. npm install
+> 1. npm run dev
+
+[3] Click on the *remotestorage* widget (logo)
+
+[4] type in `tester@localhost:8000` in the input field of the widget, click through
+
+* *tester* being a test username
+* *localhost:8000* is where the *remotestorage* server is responding as per step 2.
+
+[5] what happens at this point is server specific, OAuth flow should redirect you to onboarding and login
```

Comment:
Actually, the username and port number, i.e. user address, above is also server-specific, and usually you will have to create a user account first. Maybe it would make sense to declare this entire howto being specific to Armadietto?